lib/internal/util.js

```diff
@@ -441,6 +442,13 @@ function createDeferredPromise() {
   return { promise, resolve, reject };
 }
 
+let DOMException;
+const lazyDOMException = hideStackFrames((message, name) => {
+  if (DOMException === undefined)
+    DOMException = internalBinding('messaging').DOMException;
```

Comment:
I think it's good to hoist this out of the crypto util and make it more universal in the code base, but I am wondering, why do we want to make this lazy?

Reply:
Probably to avoid building the JS function until we're on the error path?

Reply:
@jasnell